scripts/skills/special/rf_poise_effect.nut

```diff
@@ -0,0 +1,142 @@
+/*
+Internally we calculate the current threshold inversely. That makes it much cleaner in code to handle increases and reduction of PoiseMax.
+Our intended behavior is that a change in PoiseMax also changes the Poise of that character by the same amount:
+
+The PoiseMax is purely handled in the Properties while the current Poise is handled in this skill.
+It is much cleaner when a change to PoiseMax can solely focus on changing variables in Properties.
+*/
+
+this.rf_poise_effect <- ::inherit("scripts/skills/skill", {
+	m = {
+		// Config
+		HeadshotMult = 1.5,	// multiplier for incoming poise damage on a hit to the head (unless steelbrow was learned)
+
+		// Private
+		InversePoise = 0,	// This is the inverse of Poise and counts upwards instead of downwards as Poise damaged is received
```

Comment:
I think having Poise as a character property would be better. Then this skill can use the incoming damage to reduce the Poise in the CurrentProperties of this character. This handles Poise similar on the backend to other attributes e.g. Resolve and that keeps things cleaner and keeps the character properties in one place.

Reply:
I'm unsure about putting the built-up poise in the character properties when similar stats like Hitpoints, Fatigue and Actionpoints are located in the actor.nut
Is that really the right place for a value that does not reset to a base value during any update call?

Reply:
I see. In that case it makes more sense to have it in `actor` instead of in `properties`, similar to Fatigue and Hitpoints.